doc/classes/FileAccess.xml

```diff
@@ -52,6 +52,13 @@
 		<link title="3D Voxel Demo">https://godotengine.org/asset-library/asset/676</link>
 	</tutorials>
 	<methods>
+		<method name="close">
+			<return type="void" />
+			<description>
+				Closes the currently opened file and prevents subsequent read/write operations. Use flush to persist the data to disk without closing the file.
```

Comment:
```suggestion
				Closes the currently opened file and prevents subsequent read/write operations. Use [method flush] to persist the data to disk without closing the file.
```

Reply:
Forgot about this before merging. One of us can sneak this nitpick into another unrelated bugfix or documentation commit next time :P